lib/apachetest.pm

```diff
@@ -78,6 +78,10 @@ sub setup_apache2 {
 
     # Check server certificate is available, which should be generated during apache2-mod_nss installation
     if ($mode =~ m/NSS/) {
+        if (check_var('VERSION', 15)) {
```

Comment:
why not use `is_sle && sle_version_at_least('15')`? We don't want to check the version again for '15-SP1' or for Leap, '15.1'

Reply:
I added SLE as well as Leap provision for versions 15+.